containerz/containerz.proto

```diff
@@ -299,3 +327,69 @@ message LogRequest {
 message LogResponse {
   string msg = 1;
 }
+
+message CreateVolumeRequest {
+  // name of the volume. If present, this system must create the volume with
+  // this name otherwise it must allocate a name.
+  string name = 1;
+
+  // The volume driver. If not specified, this defaults to the the standard
+  // linux mount operation (http://man7.org/linux/man-pages/man8/mount.8.html)
+  // and associated options.
+  string driver = 2;
+
+  // Options for the driver. The actual option keys and values are driver
+  // specific.
+  map<string, string> options = 3;
+
+  // Labels to apply to the volume. Labels are metadata for the the volume.
+  map<string, string> labels = 4;
+}
+
+message CreateVolumeResponse {
+  // The name of the created volume.
+  string name = 1;
+}
+
+message RemoveVolumeRequest {
+  // The volume to remove.
+  string name = 1;
+}
+
+message RemoveVolumeResponse {
+}
+
+message ListVolumeRequest {
+  // List all volumes.
+  bool all = 1;
+
+  // Only return limit number of volumes.
+  int32 limit = 2;
+
+  // A filter is a mapping from a key to a list of values.
```

Comment:
Is this a requirement today (and do you expect different backing container runtimes to support this same filtering)?

Reply:
See comment above